src/index.js

```diff
@@ -0,0 +1,49 @@
+// Imports
+import './style.css';
+
+// variables
+const list = [
+  {
+    description: 'some description',
+    completed: 0,
+    index: 0,
+  },
+  {
+    description: 'today\'s task',
+    completed: 0,
+    index: 0,
+  },
+  {
+    description: 'yesterday\'s tast description',
+    completed: 0,
+    index: 0,
+  },
```

Comment:
- [ ] On page load, the list array should be rendered dynamically to the DOM please work on that

Reply:
Hello @DeliceLydia , thanks for your feedback
It looks that I broke the code when I was fixing the linters errors in commit 9658a201f6ae313a2c41c1a189f149f980c4c8f6
https://github.com/MacCrazyman/to-do-list/blob/9658a201f6ae313a2c41c1a189f149f980c4c8f6/src/index.js#L24
https://github.com/MacCrazyman/to-do-list/blob/85de91fd54caad9c2b6a128693eaa63ef7c74c79/src/index.js#L24